src/dmd/root/man.d

```diff
@@ -33,7 +33,7 @@ version (Windows)
 }
 else version (OSX)
 {
-    extern (C++) void browse(const(char)* url)
+    extern (C++) void browse(const(char)* url) nothrow @nogc
```

Comment:
Other versions?

Reply:
Good, point. Added, let's see what the CIs are saying.